src/gui/main_window.py

```
@@ -3,7 +3,7 @@
 from .order_widget import OrderWidget
 from .server_widget import ServerWidget
 import asyncio
-from ..websocket.client import WebSocketClient
+from ..supabase_client import SupabaseClient
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)